### PR TITLE
Add container for running tempest tests

### PR DESCRIPTION
**Pull request by craigbr**
Commented out of docker-compose.yml so it doesn't run every
time

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,7 +1,12 @@
 version: '2'
 services:
+
   influxdb:
     image: influxdb:latest
+
+  agent:
```
Comment (mhoppal) [inline docker-compose.yml]: why not the official one?

File: docker-compose.yml [hunk 2]
```diff
@@ -1,7 +1,12 @@
...
+    image: hoppalm/monasca-agent:latest
```
Comment (mhoppal) [inline docker-compose.yml]: why not the official one

File: docker-compose.yml [hunk 3]
```diff
@@ -60,7 +71,8 @@ services:
     image: timothyb89/monasca-sidecar:1.0.0
 
   monasca:
-    image: monasca/api:master-prometheus
+    # image: monasca/api:master-prometheus
+    image:  timothyb89/monasca-api:prometheus-test-20170519-172008
```
Comment (mhoppal) [inline docker-compose.yml]: did we not push this to monasca?

File: docker-compose.yml (hunk 2 above)
Comment (mhoppal) [inline docker-compose.yml]: Also just out of curiosity does this have the docker plugin running?

File: docker-compose.yml [hunk 4]
```diff
@@ -60,7 +65,7 @@ services:
     image: timothyb89/monasca-sidecar:1.0.0
 
   monasca:
-    image: monasca/api:master-prometheus
+    image: monasca/api:latest
```
Comment (timothyb89) [inline docker-compose.yml]: `monasca/api:master-20170524-151708` was just pushed and would be the correct API image to use

File: tempest-tests/README.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+ * `latest`: refers to the latest tempest-tests container
+
+Usage
+-----
+
+To run, this container needs to connect to a working Keystone and Monasca API.
+
+In environments resembling the [official Kubernetes environment]
+does not require additional configuration parameters, and can be run like so:
+
```
Comment (mhoppal) [inline tempest-tests/README.md]: run like what?

File: tempest-tests/README.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+To run, this container needs to connect to a working Keystone and Monasca API.
+
+To run in environments resembling the [official Kubernetes environment][1] see
+the [Monasca HELM charts][2].
+
+In order to run in docker-compose add this section to docker-compose.yaml:
+```
+  tempest-tests:
+    image: monasca/tempest-tests:master
```
Comment (timothyb89) [inline tempest-tests/README.md]: should `master` should probably be `latest` here?

**Pull request merged**